src/sdk/flyover.ts

```diff
@@ -589,4 +576,14 @@ export class Flyover implements Bridge {
       btcConnection: this.bitcoinDataSource
     }
   }
+
+  async hashPeginQuote (quote: Quote): Promise<string> {
```

Comment:
I'd still call `this.checkLbc()` even if you're using `assertTruthy` because `checkLbc` initializes the LBC if its not present

Reply:
same for the pegout version

Reply:
Changed.